Makefile

```diff
@@ -405,7 +405,7 @@ build-driver-adapters-kit: build-driver-adapters
 
 build-driver-adapters: ensure-prisma-present
 	@echo "Building driver adapters..."
-	@cd ../prisma && pnpm --filter "*adapter*" i
+	@cd ../prisma && pnpm i
```

Comment:
Is this a needed change? I imagine it'll increase the CI time

Reply:
Yes, it was failing even after updating to pnpm 9 in this PR precisely because of this.

Reply:
(and it was also the reason i couldn't reproduce locally, because i already had all packages installed in the local checkout)